src/components/home/Service.js

```diff
@@ -1,7 +1,28 @@
+"use client";
 import React from "react";
-
+import { HiCodeBracket } from "react-icons/hi2";
 const Service = () => {
```

Comment:
Service takes in props, Check you issue for what props to pass in. Don't hard code the words.